appengine/urlfetch/requests/main.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app]
+import logging
+
+from flask import Flask
+
+# [start imports]
+import requests
+import requests_toolbelt.adapters.appengine
+# [END imports]
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    # [START requests_get]
+    # Use the App Engine Requests adapter. This makes sure that Requests uses
+    # URLFetch.
+    requests_toolbelt.adapters.appengine.monkeypatch()
```

Comment:
I tried, but the logic hits the urlfetch service which does not have a stub until the tests initialize them. 

Actually I have a solution. Will fix.